core/server/data/migrations/versions/3.43/01-fix-backup-content-permission-typo.js

```diff
@@ -0,0 +1,15 @@
+const logging = require('../../../../../shared/logging');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(async function up(knex) {
+    const typoedPermisson = await knex.select('*').from('permissions').where('action_type', 'backupContect').first();
+
+    if (!typoedPermisson) {
+        return logging.warn('Not updating permissions, no typo found');
+    }
+
+    logging.info('Updating permissions, fixing typo by renaming "backupContect" to "backupContent"');
+    await knex('permissions').update('action_type', 'backupContent').where('action_type', 'backupContect');
+}, function down() {
```

Comment:
@allouis No biggie, but this should be updated to `async function down()` for consistency with rest of our no-ops. :)